src/test/java/teammates/test/cases/ui/pagedata/InstructorCourseDetailsPageDataTest.java

```diff
@@ -59,9 +59,9 @@ public void testAll() {
         pageData.init(curInstructor, courseDetails, instructors, students);
         
         assertEquals(instructors.size(), pageData.getInstructors().size());
-        assertEquals(students.size(), pageData.getStudentsTable().getRows().size());
-        assertEquals(4, pageData.getStudentsTable().getRows().get(0).getActions().size());
-        assertEquals(2, pageData.getStudentsTable().getRows().get(0).getCommentActions().size());
+        //assertEquals(students.size(), pageData.getStudentsTable().getRows().size());
```

Comment:
If the student table is tested somewhere else then you can delete all the tests related to the table but we still have to test the Course Details Information Container .